tensorflow/core/transforms/utils/utils.cc

```diff
@@ -42,6 +42,13 @@ bool NodeIsOnCpu(Operation *op) {
          absl::StartsWith(device, tensorflow::DEVICE_CPU);
 }
 
+bool NodeIsOnGpu(Operation *op) {
```

Comment:
nit: I'd refactor a bit into
NodeIsOnGpu, calls into NodeIsOnDevice(op,  tensorflow::DEVICE_GPU)
NodeIsOpCpu, calls into NodeIsOnDevice(op, tensorflow::DEVICE_CPU)

static to the utils.cc - NodeIsOnDevice(Operation, DeviceName)

Reply:
Done